packages/plugin-core/src/commands/DiagnosticsReport.ts

```diff
@@ -32,11 +33,18 @@ export class DiagnosticsReportCommand extends BasicCommand<
       path.dirname(logPath),
       "dendron.server.log"
     );
-    const serverLogFile = fs.readFileSync(serverLogPath, { encoding: "utf8" });
-    const serverLastLines = serverLogFile.slice(-3000);
+
+    const isDevMode = VSCodeUtils.isDevMode();
+    let serverLastLines: string = "";
+    if (!isDevMode) {
```

Comment:
instead of checking for mode, why don't you check if the file exists? if so, then read it